mlog/global.go

```diff
@@ -20,7 +22,7 @@ func InitGlobalLogger(logger *Logger) {
 }
 
 func RedirectStdLog(logger *Logger) {
-	zap.RedirectStdLogAt(logger.zap.With(zap.String("source", "stdlog")), zapcore.ErrorLevel)
+	zap.RedirectStdLogAt(logger.zap.With(zap.String("source", "stdlog")).WithOptions(zap.AddCallerSkip(-2)), zapcore.ErrorLevel)
```

Comment:
Why negative `2` here?

Reply:
Because it's the magic number that makes it work :P